mission_control/navigator_launch/launch/gnc.launch

```diff
@@ -44,30 +40,25 @@
         <rosparam param="effort_ratio">1.35</rosparam><!-- map 220 newtons to 300 effort units -->
         <rosparam param="effort_limit">400</rosparam>
     </node>
-    
+
     <include file="$(find navigator_launch)/launch/tf.launch"/>
 
-    <node pkg="sub8_alarm" type="alarm_handler.py" name="alarm_handler"/>
-    
+    <!-- Alarms handlers -->
+    <node pkg="sub8_alarm" type="alarm_handler.py" name="alarm_handler"/>  <!-- TODO: Remove this dependacy, without destroying history -->
+    <node pkg="navigator_launch" type="network_listener.py" name="network_kill"/>
+
     <node name="wrench_arbiter" pkg="navigator_msg_multiplexer" type="wrench_arbiter.py"/>
 
     <node name="pose_controller" pkg="navigator_controller" type="mrac_controller.py"/>
     <!-- can be type="mrac_controller.py" OR type="run_nn_controller" if you have that dependency
     <https://github.com/jnez71/neural_control>-->
 
-    <node if="$(arg rc)" pkg="navigator_joystick_control" type="navigator_joystick.py" name="joystick_wrench" output="screen">
+    <node pkg="navigator_joystick_control" type="navigator_joystick.py" name="joystick_wrench" output="screen">
```

Comment:
Maybe for a future PR it would be nice to have these parameters be dynamic_reconfigure(able) as torque and force sensitivities. Haha it would be really fun to have the bumpers temporarily improve sensitivity, BOOST!